arch/arm/boot/dts/bcm2710-rpi-2-b.dts

```diff
@@ -0,0 +1,125 @@
+/dts-v1/;
+
+#include "bcm2710.dtsi"
+#include "bcm2709-rpi.dtsi"
+#include "bcm283x-rpi-smsc9514.dtsi"
+#include "bcm283x-rpi-csi1-2lane.dtsi"
+
+/ {
+	compatible = "raspberrypi,2-model-b", "brcm,bcm2836";
+	model = "Raspberry Pi 2 Model B";
```

Comment:
This should be changed represent the real SoC and maybe a different board naming to distinguish the Raspberry Pi 2 boards.